embark/uploader/views.py

```diff
@@ -328,16 +341,21 @@ def get_load(request):
 
 @csrf_exempt
 @require_http_methods(["GET"])
-def get_individual_report(request):
-    firmware_id = request.GET.get('id', None)
+def get_individual_report(request, analyze_id):
+    firmware_id = analyze_id
     if not firmware_id:
         logger.error('Bad request for get_individual_report')
         return JsonResponse(data={'error': 'Bad request'}, status=HTTPStatus.BAD_REQUEST)
     try:
         result = Result.objects.get(firmware_id=int(firmware_id))
-        result = model_to_dict(result)
-        result['strcpy_bin'] = json.loads(result['strcpy_bin'])
-        return JsonResponse(data=result, status=HTTPStatus.OK)
+        firmware = Firmware.objects.get(pk=int(firmware_id))
```

Comment:
maybe rename this, because otherwise the naming is confusing (firmware.firmware)

Reply:
Done !